### ISSUE TITLE
Not seeing progress bar

### ISSUE THREAD

**Issue by rscorer** [inertiajs/inertia#1408]
### Version:

- `@inertiajs/vue3` version: 1.0.0

### Describe the problem:

I have a 5 second delay in my backend code, and the previous versions of inertia correctly showed a progress bar.

I have upgraded to 1.0.0 and no longer see any progress bar.

### Steps to reproduce:

Create a Page, called Contact.vue. In the backend controller inject a 5 second delay before responding with the inertia render call... now go to the URL which will display the contact page and see nothing for 5 seconds, then the contact page,,,

**Comment by rscorer** [comment]
I also tried the workaround mentioned in the Progress Indicators documentation, by adding NProgress manually, and that also did not show a progress bar.

**Comment by nbc38** [comment]
Didn't work for me too.
I just went to the legacy documentation and followed the steps and it works. 
Maybe there is something missing in the new docs?

**Comment by cbaconnier** [comment]
Have you upgraded the progress bar as well?

https://inertiajs.com/upgrade-guide#progress

```bash
npm remove @inertiajs/progress
```

```diff
- import { InertiaProgress } from '@inertiajs/progress'

- InertiaProgress.init()
```

```javascript
createInertiaApp({
  progress: {
    color: '#29d',
  },
  // ...
})
```

**Comment by rscorer** [comment]
Yes, I removed the old progress imports and code, tried `createInertiaApp` with and without the progress setting.

All to no avail 

```
import { createApp, h } from 'vue'
import { createInertiaApp } from '@inertiajs/vue3'

createInertiaApp({
    resolve: name => {
        const pages = import.meta.glob('./Pages/**/*.vue', { eager: true })
        return pages[`./Pages/${name}.vue`]
    },
    progress: {
        color: '#29d',
    },
    setup({ el, App, props, plugin }) {
        createApp({ render: () => h(App, props) })
            .use(plugin)
            .mount(el)
    },
})
```

**Comment by rscorer** [comment]
> Didn't work for me too.
> I just went to the legacy documentation and followed the steps and it works.
> Maybe there is something missing in the new docs?

Could be!

**Comment by reinink** [comment]
Really hard to guess at why this isn't working without seeing a minimal reproduction. Please provide a *minimal* reproduction as a Git repo 👍

**Comment by rscorer** [comment]
I'm using the community go adapter, which hasn't been updated since September last year - will it need updating now inertiaJS is 1.0? That seems most likely IMHO. I can try and put together a simpler reproduction and post it tomorrow if that works
